Source/WebKit/WebProcess/InjectedBundle/InjectedBundle.cpp

```diff
@@ -361,11 +361,4 @@ void InjectedBundle::setTabKeyCyclesThroughElements(WebPage* page, bool enabled)
     page->corePage()->setTabKeyCyclesThroughElements(enabled);
 }
 
-void InjectedBundle::setAccessibilityIsolatedTreeEnabled(bool enabled)
```

Comment:
Seems like this needs to be removed from the header too, right?

Reply:
Fixed, thanks!